.github/workflows/backend.yml

```diff
@@ -105,6 +105,58 @@ jobs:
       - name: Handle artifacts
         uses: ./.github/actions/artifacts
 
+  backend-test-snuba-contains-metrics-tag-values:
```

Comment:
if this is a required check it should get added to the `needs` for the collector job at the bottom

Reply:
I wouldn't want to make this a required check and block all teams from shipping in case I messed something up.